### PR TITLE
feat: VAN-666 - Reject new password that is detected as vulnerable

### PR DESCRIPTION
**Pull request by shafqatfarhan**
[VAN-666](https://openedx.atlassian.net/browse/VAN-666)

### CODE REVIEW

File: src/reset-password/ResetPasswordPage.jsx [hunk 1]
```diff
@@ -53,7 +53,11 @@ const ResetPasswordPage = (props) => {
   const validatePasswordFromBackend = async (password) => {
     let errorMessage = '';
     try {
-      errorMessage = await validatePassword(password);
+      const payload = {
+        reset_password_page: true,
```
Comment (waheedahmed) [inline src/reset-password/ResetPasswordPage.jsx:57]: Do we still need this since we have a separate feature flag at the backend?

**Review by waheedahmed — APPROVED** [review]
One question otherwise 👍

File: src/reset-password/ResetPasswordPage.jsx (hunk 1 above)
Comment (shafqatfarhan) [reply src/reset-password/ResetPasswordPage.jsx:57]: yes, this is required along with the feature flag to enable validation only for reset password page

Comment (shafqatfarhan) [reply src/reset-password/ResetPasswordPage.jsx:57]: We are using the same Validation API endpoint for registration and sign-in validations as well

**Pull request merged**